pom.xml

```diff
@@ -101,6 +101,24 @@
             <version>0.100.1.0</version>
             <scope>provided</scope>
         </dependency>
+        <dependency>
+            <groupId>com.cjcrafter</groupId>
+            <artifactId>weaponmechanics</artifactId>
+            <version>4.1.0</version>
+            <scope>provided</scope>
+        </dependency>
+        <dependency>
+            <groupId>com.cjcrafter</groupId>
+            <artifactId>mechanicscore</artifactId>
+            <version>4.1.0</version>
+            <scope>provided</scope>
+        </dependency>
+        <dependency>
+            <groupId>com.jeff_media</groupId>
+            <artifactId>SpigotUpdateChecker</artifactId>
```

Comment:
It looks like SpigotUpdateChecker is not for updating your dependencies but rather to remind server owners to update your plugin. I think it probably shouldn't be included in this PR (although maybe I'll add it some other time because it probably would be useful)